Solutions/Box/Analytic Rules/BoxMultipleItemsDeletedByUser.yaml

```diff
@@ -22,11 +22,10 @@ query: |
   | where EventType =~ 'DELETE'
   | summarize deleted_items = dcount(SourceItemName) by SrcUserName, bin(TimeGenerated, 5m)
   | where deleted_items > threshold

```

Comment:
same comment above.. Add output formatting, aggregation and additional entitytypes along with UPNDomain